css/styles.css

```diff
@@ -0,0 +1,343 @@
+/**Global Colors & Fonts**/
+:root{
+ --color-white: #ffffff;
+ --color-navy: #093c60;
+ --color-grey: #ececec;
+ --color-green: #81b741;
+ --color-black: #000000;
+
+ --title-font:'Vollkorn', serif;
+ --body-font:'Montserrat', sans-serif;
+}
+
+
+
+/**layout**/
+.row{
+  margin-bottom: 2rem;
+}
+main{
+  max-width: 68.75rem;
+  margin: auto;
+  padding-left:0.75rem;
+  padding-right:0.75rem;
+}
+.d-flex{
```

Comment:
If you import bootstrap you won't need to do this. It's part of bootstrap.

Reply:
I ended up removing bootstrap and trying to code it myself to practice.